ts/features/wallet/paypal/component/PayPalCheckoutPspComponent.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { ListItem, View } from "native-base";
+import { StyleSheet } from "react-native";
+import { H3 } from "../../../../components/core/typography/H3";
+import { ImportoEuroCents } from "../../../../../definitions/backend/ImportoEuroCents";
+import { H4 } from "../../../../components/core/typography/H4";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { Label } from "../../../../components/core/typography/Label";
+import IconFont from "../../../../components/ui/IconFont";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import I18n from "../../../../i18n";
+import { Link } from "../../../../components/core/typography/Link";
+import { openWebUrl } from "../../../../utils/url";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: "column"
+  },
+  pspContainer: {
+    alignItems: "flex-start",
+    flexDirection: "column"
+  },
+  pspHeader: {
+    flexDirection: "row"
+  },
+  icon: {
```

Comment:
This style seems unused

Reply:
Yes it does!
Removed here https://github.com/pagopa/io-app/pull/3559/commits/13063366908596b3629fb4b91b3b7232d11bdf43